docs/3.api/1.components/12.google-maps.md

```diff
@@ -0,0 +1,64 @@
+---
+title: "<GoogleMaps>"
+description: A simple and performant Google Maps component.
+links:
+  - label: Source
+    icon: i-simple-icons-github
+    to: https://github.com/nuxt/scripts-and-assets/blob/main/modules/nuxt-third-party-capital/src/runtime/components/GoogleMaps.ts
+    size: xs
+---
+
+The GoogleMaps component feels like a Google Maps Embed, but uses the [Maps JavaScript API](https://developers.google.com/maps/documentation/javascript) to load and initialize the map.
+
+::callout
+You need an api key to use Google Maps.
+::
+
+## Params
+
+An object containing the following options:
+
+| name | type | description                       |
+|:-----|:-------|:--------------------------------|
+| apiKey | string | The [api key](https://developers.google.com/maps/documentation/javascript/get-api-key) needed to use Google Maps API. (required)|
+| q | string | A query to search for. (optional)|
+| center | LatLng | Coordinates to set the map to. (optional) |
+| zoom | number | Zoom value for the map. (optional) |
+| width | string | Width of the player. (optional)|
+| height | string | Height of the player. (optional)|
+
+Either a query (q) or coordinates (center) are needed for the map to function properly.
+
+::callout
+Camel-case prop names are assigned kebab-cased in DOM templates (see [Prop name casing](https://vuejs.org/style-guide/rules-strongly-recommended.html#prop-name-casing)).
```

Comment:
I'll let Julien, Daniel, or Harlan chime in here, but if this is a common convention than I'll consider removing this. Not sure its necessary to include if this is how things are typically done.

Reply:
I'll remove